foursight_core/scripts/decrypt_accounts_file.py

```diff
@@ -0,0 +1,52 @@
+import io
+import sys
+from foursight_core.react.api.encryption import Encryption
+
+ACCOUNTS_FILE = "foursight_core/accounts.json"
+
+
+def decrypt_accounts_file(accounts_file = ACCOUNTS_FILE, password = None):
+
+    encryption = Encryption(password)
+
+    with io.open(accounts_file) as accounts_f:
+        accounts_content = accounts_f.read()
+        if accounts_file.endswith("accounts.json") and accounts_content.startswith("["):
+            print(f"File seems to already be decrypted: {accounts_file}")
+            exit(1)
+        accounts_content_decrypted = encryption.decrypt(accounts_content)
```

Comment:
Putting a context manager around so much code makes it look like the accounts_f might be used more than once. I think this would be clearer(untested, relying on importing `file_contents` from `dcicutils.misc_utils`):

```suggestion
    accounts_content = file_contents(accounts_f)
    if accounts_file.endswith("accounts.json") and accounts_content.startswith("["):
        print(f"File seems to already be decrypted: {accounts_file}")
        exit(1)
    accounts_content_decrypted = encryption.decrypt(accounts_content)
```
But if the file might contain binary and you're opening it like it's text, is there a possibility "[" could just be the first couple of bytes of the binary?  It should at least mention it's heuristic. This is why I don't understand not using a different extension.

Reply:
Yeah, I'm going to take your suggestion of distinguishing encrypted vs non-encrypted via file name suffix.